rewrite-java/src/main/java/org/openrewrite/java/JavaRefactorVisitor.java

```diff
@@ -451,24 +452,57 @@ public J visitWildcard(J.Wildcard wildcard) {
         return w.withBoundedType(refactor(w.getBoundedType()));
     }
 
+    /**
+     * This method will add an import to the compilation unit.
+     *
+     * @param fullyQualifiedName Fully-qualified name of the class.
+     */
+    public void addImport(String fullyQualifiedName) {
+        AddImport op = new AddImport();
+        op.setType(fullyQualifiedName);
+        op.setOnlyIfReferenced(false);
+        if (!andThen().contains(op)) {
+            andThen(op);
+        }
+    }
```

Comment:
This JavaDoc comment conveys very little that isn't already in the method signature itself.
Such comments are of marginal use since they restate the obvious, but in a way the compiler cannot assist in verifying. 

However, while this method is short and generally well described by its signature alone, its implementation has several implications that are not obvious. Such as:

* Being implemented via `andThen(AddImport)` means that nothing will have changed about the compilation unit immediately after calling the method
* It will add the import even if the symbol in question is not referenced anywhere. This _can_ be inferred from the existence of `maybeAddImport()` but being explicit in the javadoc wouldn't hurt
* It's safe (both in a correctness and performance sense) to call this multiple times in the same compilation unit since it checks that only one `AddImport` will ever be scheduled to run.

Is it fully necessary or desirable to spell out all of this out in the comment? Probably not, but I list them to provide examples of the sorts of things good JavaDoc comments describe.

Reply:
I think that since this is likely a core API call that will be used by a ton of visitors, its probably best to error on the side of over-explaining. I like the idea of explaining the implications.

Reply:
in fact, now that you enumerated (what sound like obvious statements), it makes me think that calling this would be marginally more efficient in my use case because I KNOW that there is a reference to said import, no need to "try and figure it out"